create.js

```diff
@@ -2,6 +2,12 @@ module.exports = create
 
 var Buffer = require('buffer').Buffer
 
+var majorVersion = process.version.replace(/^v/, '').split('.')[0];
+
 function create(size) {
-  return Buffer.alloc(size)
+  if (!majorVersion || majorVersion < 6) {
+    return new Buffer(size)
+  } else {
+    return Buffer.alloc(size)
+  }
```

Comment:
Would you mind moving this to the outside of the `create` function? Something like:

```suggestion
var version = ((process || {}).version || 'v0.0.0').slice(1).split('.')[0]
var create = (
  Number(version) < 6
  ? function create (size) { return new Buffer(size) }
  : function create(size) { return Buffer.alloc(size) }
)
```

Reply:
👍